PhysicsTools/PatAlgos/plugins/SlimCaloJetSelector.h

```diff
@@ -0,0 +1,84 @@
+#ifndef PhysicsTools_PatAlgos_SlimCaloJetSelector_h
+#define PhysicsTools_PatAlgos_SlimCaloJetSelector_h
+
+#include "FWCore/Framework/interface/stream/EDFilter.h"
+#include "DataFormats/Common/interface/RefVector.h"
+
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/ObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleElementCollectionSelector.h"
+
+#include "DataFormats/JetReco/interface/CaloJet.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include <vector>
+
+class SlimCaloJetSelector : public edm::stream::EDFilter<> {
```

Comment:
Not only naming convention but also contents is not in line with other slimmers.
Output is not really PAT object but RECO object. So as far as I understand, it can not be simply PATCaloJetSlimmer. 

Taejeong